src/store/modules/wallets/wallets-plugin.ts

```diff
@@ -0,0 +1,64 @@
+import VuexPersistence from 'vuex-persist'
+import { mapWallets } from '@/store/modules/wallets/utils'
+import { WalletsState } from '@/store/modules/wallets/types'
+import { TypedStorage } from '@/lib/typed-storage'
+
+const WALLETS_STATE_STORAGE_KEY = 'WALLETS_STATE_STORAGE'
+
+const stateStorage = new TypedStorage<typeof WALLETS_STATE_STORAGE_KEY, WalletsState | null>(
+  WALLETS_STATE_STORAGE_KEY,
+  null,
+  window.localStorage
+)
+function initWallets() {
+  let state = stateStorage.getItem()
+
+  if (!state) {
+    state = { symbols: mapWallets() }
+    stateStorage.setItem(state)
+  } else {
+    const initialTemplate = mapWallets()
+
+    const hasDifference = !!initialTemplate.filter(
+      ({ symbol: symbol1 }) => !state!.symbols.some(({ symbol: symbol2 }) => symbol2 === symbol1)
+    ).length
+
+    if (hasDifference) {
+      state.symbols = mapWallets()
+      stateStorage.setItem(state)
+    }
+  }
```

Comment:
You call `mapWallets()` in three places.

```suggestion
  const state = stateStorage.getItem()
  const defaultState = { symbols: mapWallets() }

  if (!state) {
    stateStorage.setItem(defaultState)
  } else {
    const hasDifference = !!initialTemplate.filter(
      ({ symbol: symbol1 }) => !state!.symbols.some(({ symbol: symbol2 }) => symbol2 === symbol1)
    ).length

    if (hasDifference) {
      stateStorage.setItem(defaultState)
    }
  }
```